packages/wallets/src/hooks/useDevice.tsx

```diff
@@ -0,0 +1,16 @@
+import { useWindowSize } from 'usehooks-ts';
+
```

Comment:
```suggestion

```
- Should add a comment here mentioning that this should be only used for behaviour and not for hiding UI